keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -103,7 +101,6 @@ Traffic should never bounce to a daemon on another node.
 ### Risks and Mitigations
 
 * When the `Local` policy is set, it is the user's responsibility to ensure node-local endpoints are ready, otherwise traffic will be dropped.
-* Using the `Local` or `PreferLocal` policy may result in imbalanced traffic for pods in a Service. It is the user's responsibility to handle this.
 
```

Comment:
We have to add something like:

The special service `kubernetes` created by the apiserver will never have `internalTrafficPolicy` local or something like this

Reply:
That's managed by kube-apiserver, so I don't see how it's a risk? Can you expand?

Reply:
We don't disclaim every other field, I don't think we need to disclaim this one.